cmd/ui/src/views/Explore/ContextMenu/CopyMenuItemV2.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { faCaretRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { MenuItem, Tooltip, TooltipProps, styled, tooltipClasses } from '@mui/material';
+import { NodeResponse, useExploreSelectedItem, useNotifications } from 'bh-shared-ui';
+
+const StyledTooltip = styled(({ className, ...props }: TooltipProps) => (
+    <Tooltip {...props} classes={{ popper: className }} />
+))(({ theme }) => ({
+    [`& .${tooltipClasses.tooltip}`]: {
+        color: theme.palette.text.primary,
+        backgroundColor: theme.palette.background.paper,
+        padding: 0,
+        paddingTop: '0.5rem',
+        paddingBottom: '0.5rem',
+        boxShadow: theme.shadows[8],
+        marginLeft: '2px !important',
+    },
+}));
+
+const CopyMenuItemV2 = () => {
```

Comment:
Updating this component wasnt called out in any ticket and we wouldve missed it. TY TY TY for catching that!!